packages/compartment-mapper/NEWS.md

```diff
@@ -1,5 +1,12 @@
 User-visible changes to `@endo/compartment-mapper`:
 
+# v1.3.0 (2024-??-??)
```

Comment:
Please spell this `# Next release`. I have a grep in the release workflow to update all these.